compiler/modules/bir/verify.bal

```diff
@@ -315,9 +315,8 @@ function verifyEquality(VerifyContext vc, EqualityInsn insn) returns err:Semanti
     }
 }
 
-// After JBUG #17977, #32245 is fixed, replace by ==
 function isEqual(ConstOperand c1, ConstOperand c2) returns boolean {
```

Comment:
Shouldn't the function be removed altogether ?

Reply:
fixed with https://github.com/ballerina-platform/nballerina/commit/8495fef3ea1efdbe2d1dd8c19397164205b8ac1c